.stitch/designs/prd.html

```
@@ -0,0 +1,112 @@
+# SkillSwap: UI/UX Specification
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)